demos/checkmarx/README.md

```diff
@@ -0,0 +1,37 @@
+# configure checkmarx-plugin
+
```

Comment:
Can you add the URL to the plugin with something like the below suggestion?

```suggestion
Basic configuration of the [Checkmarx plugin](https://plugins.jenkins.io/checkmarx)
```

Reply:
Sure ! 
done in : 512f236

I just saw that version 8.90.4 is public now. There should be new configuration options to specify a proxy to reach the checkmarx server.
I will update the demo again on monday (from a tested and validated config ;) )